setup/artifacts/gcs.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: single
+title:  "Google Cloud Storage Artifacts"
```

Comment:
What about them? Configuring them? What buttons are we pushing here to get stages to GCS objects as artifacts, and how can we put that into one verb at the beginning of this title?

"Identifying"?

Reply:
I think "Configuring GCS Artifact Credentials" sounds better, WDYT?